front-end/src/__tests__/__snapshots__/App.test.js.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Should match the snapshot. 1`] = `"<div class=\\"App\\"><div class=\\"SideBar\\"><div class=\\"SideBar-logo\\"><div class=\\"SideBar-logo-text\\">TextSavvy</div></div><div class=\\"SideBar-divider\\"></div><div class=\\"SidebarWorkspace\\"><div class=\\"SidebarWorkspace-header\\"><span>My Workspaces</span><img alt=\\"add workspace\\" src=\\"/assets/images/add-workspace.svg\\" class=\\"SidebarWorkspace-add-icon\\"/></div></div><div class=\\"SideBar-divider-light\\"></div><div class=\\"SideBar-spacing\\"></div><div class=\\"SideBar-manageacc SideBar-option\\"><div class=\\"SideBar-option-logo\\"><img alt=\\"gear\\" src=\\"/assets/images/gear_sm.svg\\" class=\\"icon\\"/></div><div class=\\"SideBar-option-text\\">Manage Account</div></div><div class=\\"SideBar-logout SideBar-option\\"><div class=\\"SideBar-option-logo\\"><img alt=\\"logout\\" src=\\"/assets/images/logout.svg\\" class=\\"icon\\"/></div><div class=\\"SideBar-option-text\\">Logout</div></div></div><div class=\\"Dashboard\\"><div class=\\"Dashboard-title\\"></div></div></div>"`;
+exports[`Should match the snapshot. 1`] = `"<div class=\\"App\\"><div class=\\"SideBar\\"><div class=\\"SideBar-logo\\"><div class=\\"SideBar-logo-text\\">TextSavvy</div></div><div class=\\"SideBar-divider\\"></div><div class=\\"SidebarWorkspace\\"><div class=\\"SidebarWorkspace-header\\"><span>My Workspaces</span><img alt=\\"add workspace\\" src=\\"/assets/images/add-workspace.svg\\" class=\\"SidebarWorkspace-add-icon\\"/></div></div><div class=\\"SideBar-spacing\\"></div><div class=\\"SideBar-divider-light\\"></div><div class=\\"SideBar-manageacc SideBar-option\\"><div class=\\"SideBar-option-logo\\"><img alt=\\"gear\\" src=\\"/assets/images/gear_sm.svg\\" class=\\"icon\\"/></div><div class=\\"SideBar-option-text\\">Manage Account</div></div><div class=\\"SideBar-logout SideBar-option\\"><div class=\\"SideBar-option-logo\\"><img alt=\\"logout\\" src=\\"/assets/images/logout.svg\\" class=\\"icon\\"/></div><div class=\\"SideBar-option-text\\">Logout</div></div></div><div class=\\"Dashboard\\"><div class=\\"Dashboard-title\\"></div></div></div>"`;
```

Comment:
Not sure whether to keep these changes or the develop's changes in merge conflict.